clusters/nerc-ocp-obs/kustomization.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: openshift-gitops
+
+resources:
+  - ../lib/cluster-scope
+  - ../lib/logging
+  - ../lib/ceph-exporter
```

Comment:
Let's get [this PR merged with the cluster-scope for the obs cluster](https://github.com/OCP-on-NERC/nerc-ocp-apps/pull/36), then merge the `logging` and `loki` resources, but probably not `ceph-exporter`.